unattended_installer/install_functions/indexer.sh

```diff
@@ -161,17 +161,21 @@ function indexer_install() {
 function indexer_startCluster() {
 
     retries=0    
+    set -x
```

Comment:
Remove this

Reply:
Done in [c07a505](https://github.com/wazuh/wazuh-packages/pull/1488/commits/c07a5050dae2ca4194318489caea40608d04e38a)